readme.adoc

```diff
@@ -10,3 +10,38 @@ Furthermore, it provides _++include::[]++_ directives for embedding the results
 
 For more information on jQAssistant see https://jqassistant.org[^].
 
+== Using the jqassistant-asciidoc-report-plugin
+
+NOTE:: Starting from jqassistant-rdbms-plugin v2.0.0, the AsciiDoc-Report-Plugin resides under jqassistant-plugin.
```

Comment:
In wiefern ist das RDBMS Plugin hier relevant?

Reply:
🤦 Copy paste Fehler. Ist gefixt